src/components/RigidBody.vue

```diff
@@ -1,64 +1,97 @@
 <script setup lang="ts">
 import { type TresObject, useLoop } from '@tresjs/core'
-import { shallowRef, watch } from 'vue'
-import { useRapierContext } from '../composables/useRapier'
-import { createCollidersFromChildren } from '../utils/collider'
-import { createRigidBody } from '../utils/rigid-body'
-import type { CreateColliderReturnType } from '../types/collider'
-import type { CreateRigidBodyReturnType, RigidBodyProps } from '../types/rigid-body'
+import { Object3D } from 'three'
+import { defineExpose, onUnmounted, onUpdated, provide, shallowRef, watch } from 'vue'
```

Comment:
defineExpose is a macro, we don't need to import it from vue :)

Reply:
Yes... I even didn't notice this import, I have an auto importer with my VScode